### ISSUE TITLE
Gradle script kotlin not valid file `this::method`

### ISSUE THREAD

**Issue by JLLeitschuh** [shyiko/ktlint#30]
I have a file with this code:

```kotlin
fun repositoryConfigurer(repoHandler : RepositoryHandler) {
    repoHandler.apply {
        // ... ect...
    }
}

var configureRepository : (RepositoryHandler) -> Unit by extraWrapped
configureRepository = this::repositoryConfigurer // Fails on this line
```
I get the following error:
```
/Users/jonathanleitschuh/work/git/plexxicontrol/sharedValues.gradle.kts:138:27: Not a valid Kotlin file (unexpected tokens (use ';' to separate expressions on the same line))
```
I'm using version: `"0.3.0"`

**Comment by JLLeitschuh** [comment]
This also exists in `0.3.1`

**Comment by shyiko** [comment]
@JLLeitschuh bound callable references (::) are coming in Kotlin 1.1, while ktlint 0.3.1 depends on 1.0.6. I bumped kotlin to 1.1.M04 and released ktlint 0.4.0 ([release notes](https://github.com/shyiko/ktlint/blob/master/changelog.md#040---2017-02-01)). It should become available within 20 minutes or so (Maven Central sync).

:tada:

**Comment by JLLeitschuh** [comment]
Let me just say, I continue to be astounded by your response time.

**Comment by JLLeitschuh** [comment]
Works like a charm!
